pkg/sysregistriesv2/system_registries_v2.go

```diff
@@ -45,6 +45,7 @@ const AuthenticationFileHelper = "containers-auth.json"
 // Endpoint describes a remote location of a registry.
 type Endpoint struct {
 	// The endpoint's remote location.
+	// DEPRECATED in favor of FindRegistry / PullSourcesFromReference.
```

Comment:
(Maybe something like `Deprecated: Use FindRegistry / PullSourcesFromReference instead of accessing/interpreting data directly`, in both places.)